jwst/ami/ami_average.py

```diff
@@ -1,8 +1,7 @@
-from __future__ import division
-
 #
 #  Module for averaging LG results for a set of AMI exposures
 #
+from __future__ import division
```

Comment:
Now that we're only support py3.x environments, we can remove all the `from __future__ import ...` statements, right?